core/src/main/java/hudson/EnvVars.java

```diff
@@ -73,7 +73,7 @@
  * @author Kohsuke Kawaguchi
  */
 public class EnvVars extends TreeMap<String, String> {
-    private static final long serialVersionUID = 4320331661987259022L;
+    private static final long serialVersionUID = 1L;
```

Comment:
I'm not sure it's safe to change an existing serial version UID.

Reply:
restored the previous values